bundle/edu.gemini.qpt.client/src/main/java/edu/gemini/qpt/core/listeners/LimitsListener.java

```diff
@@ -78,7 +88,28 @@ public void propertyChange(PropertyChangeEvent evt) {
 
 			}
 
-            // LCH Laser Shutter Warnings
+			// Target timing window
+			final TimingWindowSolver timingWindowSolver = new TimingWindowSolver(a.getObs());
+			final Iterator<Interval> timingWindowIntervals = timingWindowSolver.solve(tbn.getStartTime(), tbn.getEndTime()).iterator();
+
+			if (timingWindowIntervals.hasNext()) {
+			    final Interval timingWindowInterval = timingWindowIntervals.next();
```

Comment:
These `next`s are provisional until we decide what to do with multiple intervals of constraints.

Reply:
I would say list them all: (t0 - t1), (t2 - t3),  ...  Usually it will be just one but if there are multiple instances they should probably be displayed.

Reply:
Perhaps if the interval starts and/or ends at the twilight bounds it should say "twilight" instead of the actual time?